### DISCUSSION TITLE
Shopware recipe: Apps

### DISCUSSION THREAD

**Discussion by carstenjaksch** [deployphp/deployer#3949] [Help needed]
Hi,

Shopware 6 has a new concept called apps. These are plugins that cannot be installed via Composer, but are part of the file `var/plugins.json`.

That's an issue, because the app is not part of the codebase and therefore the build process cannot happen.

I get the following error during build without db:

```
ERROR in trusted-shops-easy-integration-s6
Module not found: Error: Can't resolve '/Users/USER/Shopware/shops/custom/apps/TrustedShopsEasyIntegrationS6/Resources/app/storefront/src/main.js'
```

As you can see, Shopware tries to build assets for the app, but it is not available in the repository.

As always, information about this topic is very rare in the Shopware community. Do some folks here use the Shopware recipe and apps? How do you solve this problem?

**Comment by null93** [comment]
Just throwing out an idea without testing it: can you sync the apps from your live server to your local machine just like the recipe does for `plugins.json` and `files/theme-config`? Checkout https://github.com/deployphp/deployer/blob/master/recipe/shopware.php#L170

**Reply by carstenjaksch** [reply]
Thank you for your reply. Apps can now be installed using composer – some app developers are unaware of that and write on their store page that they can't. Issue solved. 😄
